easybuild/easyconfigs/g/gompi/gompi-2016.12.eb

```diff
@@ -0,0 +1,20 @@
+easyblock = "Toolchain"
+
+name = 'gompi'
+version = '2016.12'
```

Comment:
OpenMPI 2.0.0 was only released in 2016.06, so this versioning doesn't make much sense...